### PR TITLE
Prepare for v0.27.0 - update Metrics API and Log(bridge) API to RC, Metrics SDK & OTLP Exporter to Beta.

### PR DESCRIPTION
**Pull request by lalitb**
## Changes
Used a script to generate Cargo.toml and Changelog, so need to review it thoroughly, in case I missed something in the script.
Will separately add the script if it's all good later.

Please provide a brief description of the changes here.

## Merge requirement checklist

* [ ] [CONTRIBUTING](https://github.com/open-telemetry/opentelemetry-rust/blob/main/CONTRIBUTING.md) guidelines followed
* [ ] Unit tests added/updated (if applicable)
* [ ] Appropriate `CHANGELOG.md` files updated for non-trivial, user-facing changes
* [ ] Changes in public API reviewed (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,11 +27,11 @@ observability tools.
...
 | Signal/Component      | Overall Status     |
 | --------------------  | ------------------ |
-| Logs-API              | Beta*              |
+| Logs-API              | RC*                |
 | Logs-SDK              | Beta               |
 | Logs-OTLP Exporter    | Beta               |
 | Logs-Appender-Tracing | Beta               |
-| Metrics-API           | Beta               |
+| Metrics-API           | RC                 |
 | Metrics-SDK           | Alpha              |
```
Comment (cijothomas) [inline README.md:35]: Please move metrics - sdk, otlp also to Beta.

File: opentelemetry-appender-log/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 ## vNext
 
+## 0.27.0
+
+Released 2024-Nov-08
```
Comment (cijothomas) [inline opentelemetry-appender-log/CHANGELOG.md:7]:
```suggestion
Released 2024-Nov-11
```

Same everywhere, sorry for last min delays.

File: opentelemetry-otlp/examples/basic-otlp/Cargo.toml [hunk 3]
```diff
@@ -15,4 +15,4 @@ tokio = { version = "1.0", features = ["full"] }
 opentelemetry-appender-tracing = { path = "../../../opentelemetry-appender-tracing", default-features = false}
 tracing = { workspace = true, features = ["std"]}
 tracing-core = { workspace = true }
-tracing-subscriber = { workspace = true, features = ["env-filter","registry", "std"] }
\ No newline at end of file
+tracing-subscriber = { workspace = true, features = ["env-filter","registry", "std"] }
```
Comment (cijothomas) [inline opentelemetry-otlp/examples/basic-otlp/Cargo.toml:18]: unintended whitespace removals?

File: opentelemetry-appender-tracing/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,12 @@
 
 ## vNext
 
+## 0.27.0
+
+Released 2024-Nov-08
```
Comment (lalitb) [inline opentelemetry-appender-tracing/CHANGELOG.md:7]:
```suggestion
Released 2024-Nov-11
```

File: opentelemetry-jaeger-propagator/CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,12 @@
 
 ## vNext
 
+## 0.27.0
+
+Released 2024-Nov-08
```
Comment (lalitb) [inline opentelemetry-jaeger-propagator/CHANGELOG.md:7]:
```suggestion
Released 2024-Nov-11
```

File: opentelemetry-otlp/CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,15 @@
 
 ## vNext
 
+## 0.27.0
+
+Released 2024-Nov-08
```
Comment (lalitb) [inline opentelemetry-otlp/CHANGELOG.md:7]:
```suggestion
Released 2024-Nov-11
```

File: opentelemetry-proto/CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,13 @@
 
 ## vNext
 
+## 0.27.0
+
+Released 2024-Nov-08
```
Comment (lalitb) [inline opentelemetry-proto/CHANGELOG.md:7]:
```suggestion
Released 2024-Nov-11
```

File: opentelemetry-sdk/CHANGELOG.md [hunk 8]
```diff
@@ -2,6 +2,13 @@
 
 ## vNext
 
+## 0.27.0
+
+Released 2024-Nov-08
```
Comment (lalitb) [inline opentelemetry-sdk/CHANGELOG.md:7]:
```suggestion
Released 2024-Nov-11
```

File: opentelemetry-semantic-conventions/CHANGELOG.md [hunk 9]
```diff
@@ -2,6 +2,10 @@
 
 ## vNext
 
+## 0.27.0
+
+Released 2024-Nov-08
```
Comment (lalitb) [inline opentelemetry-semantic-conventions/CHANGELOG.md:7]:
```suggestion
Released 2024-Nov-11
```

File: opentelemetry-stdout/CHANGELOG.md [hunk 10]
```diff
@@ -2,6 +2,13 @@
 
 ## vNext
 
+## 0.27.0
+
+Released 2024-Nov-08
```
Comment (lalitb) [inline opentelemetry-stdout/CHANGELOG.md:7]:
```suggestion
Released 2024-Nov-11
```

File: opentelemetry-zipkin/CHANGELOG.md [hunk 11]
```diff
@@ -2,6 +2,15 @@
 
 ## vNext
 
+## 0.27.0
+
+Released 2024-Nov-08
```
Comment (lalitb) [inline opentelemetry-zipkin/CHANGELOG.md:7]:
```suggestion
Released 2024-Nov-11
```

File: opentelemetry/CHANGELOG.md [hunk 12]
```diff
@@ -2,6 +2,10 @@
 
 ## vNext
 
+## 0.27.0
+
+Released 2024-Nov-08
```
Comment (lalitb) [inline opentelemetry/CHANGELOG.md:7]:
```suggestion
Released 2024-Nov-11
```

**Review by cijothomas — APPROVED** [review]

**Pull request merged**